storage/oasis/consensus.go

```diff
@@ -23,13 +23,13 @@ import (
 //     boilerplate. Consider removing most types from this package, and
 //     using nodeapi types directly.
 type ConsensusClient struct {
-	nodeApi nodeapi.ConsensusApiLite
+	NodeApi nodeapi.ConsensusApiLite
```

Comment:
Looks like this was made public just so you can do `cc.NodeApi = ...` in a test. Not very clean; can we create an alternative constructor instead, where you pass in the NodeApi?

Same comment for the newly-public `Main.Cfg` in `analyzer/consensus/consensus.go`.

Reply:
Agreed, these kludges will be removed in favor of config file changes + custom constructors. I'm planning to add the config changes once 352 goes in

Reply:
Oh ok great.
325 is pretty much ready now; if you want to work on this PR, you can safely rebase on top of 325 even before it goes in.